src/test/java/blackjack/domain/DealerTest.java

```diff
@@ -0,0 +1,127 @@
+package blackjack.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class DealerTest {
+
+    @DisplayName("딜러의 이름은 '딜러'이다.")
+    @Test
+    void create() {
+        Dealer dealer = new Dealer();
+
+        String result = dealer.getName();
+
+        assertThat(result).isEqualTo("딜러");
+    }
+
+    @DisplayName("딜러는 카드를 한 장 받을 수 있다.")
+    @Test
+    void hitOneCard() {
+        Dealer dealer = new Dealer();
+        Card card = new Card(CardRank.EIGHT, CardShape.DIAMOND);
+
+        dealer.hit(card);
+
+        assertThat(dealer.getCards()).isEqualTo(List.of(card));
+    }
```

Comment:
`containsExactly` 를 써서 컬렉션을 검증해보죠 :)
```suggestion
    @DisplayName("딜러는 카드를 한 장 받을 수 있다.")
    @Test
    void hitOneCard() {
        Dealer dealer = new Dealer();
        Card card = new Card(CardRank.EIGHT, CardShape.DIAMOND);

        dealer.hit(card);

        assertThat(dealer.getCards()).containsExactly(card);
    }
```